qiskit_ibm_runtime/fake_provider/fake_backend.py

```diff
@@ -39,6 +38,11 @@
     decode_pulse_defaults,
 )
 
+try:
+    from qiskit.providers import QasmSimulatorPy as BasicSimulator
+except ImportError:
+    from qiskit.providers import BasicSimulator
+
```

Comment:
Sorry, my bad, didn't include the full import path:
```suggestion
try:
    from qiskit.providers.basicaer import QasmSimulatorPy as BasicSimulator
except ImportError:
    from qiskit.providers.basic_provider import BasicSimulator

```

Reply:
No worries, just noticed this too!